docker/init.sql

```diff
@@ -0,0 +1,130 @@
+# cria o usuario que usara o BD
```

Comment:
O comentario no slq é usando `/* <comentario> */`
```suggestion
/* cria o usuario que usara o BD */
```